### ISSUE TITLE
Torrents won't stay active

### ISSUE THREAD

**Issue by m0nhawk** [Wonderfall/docker-rtorrent-flood#23]
After "start" the torrents update the number of peers/seeds (which is non-zero), starts seeding, but in a minute it stops. This is happening for all the torrents I add.

I'm not sure how to debug this issue.

**Comment by JeremyMahieu** [comment]
I have the same. I started getting this after a recent upgrade of the docker image. Reverting to the old image did not fix it.
I start the torrent, it checks hash and stops after 5 seconds even before finishing the check.

I think I used to have this with a manual setup of rtorrent + flood. Going in the real GUI of flood, and stopping the torrent manually used to help. But haven't figured out how I can go in the rtorrent GUI with this docker setup.

**Comment by Poisonsting** [comment]
Completely fresh, new install. Also having this issue.
Torrents start, download for a bit, then never start ever again.

**Comment by Poisonsting** [comment]
Fixed by copying the generated .rtorrent.rc file out of the container, commenting out the following line, and mounting the file as a volume for the container:
 `schedule = untied_directory,5,5,"stop_untied=/data/.watch/*.torrent"`

**Comment by JeremyMahieu** [comment]
And does the used docker image need to be reverted? Or what is the causeof the error?

**Comment by Poisonsting** [comment]
Using the latest image. Just made that one change.

**Comment by JeremyMahieu** [comment]
Thank you, it works!!

**Comment by m0nhawk** [comment]
@Poisonsting That's a really nice catch! 🎉 

I have copied the content of the file and mount it inside `docker-compose.yaml`:

```
    volumes:
      - ./flood/.rtorrent.rc:/home/torrent/.rtorrent.rc
```

**Comment by m0nhawk** [comment]
Fixed by #26
